english/zoo/zoo.md

```diff
@@ -7,5 +7,5 @@ I head over and buy some [lemonade](../drink/lemonade/lemonade.md)
 I choose to get better at [English](../get-better-at-english/english.md)
 
 After my self trolling is complete, I decide to drink the lemonade.
-As I lift the lemonade to my mouth I see the tigers have broken loose.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.